internal/core/core.go

```diff
@@ -25,7 +28,29 @@ func BuildRootCmd(conf *config.Config) cobra.Command {
 				return err
 			}
 
-			terminal.ForceColours(conf.GlobalFlags.Colors)
+			terminal.ForceColours(conf.GlobalFlags.Colors && terminal.IsStdoutTerminal())
```

Comment:
Colours: there is a wider problem here with them. Somewhere along the way it seems that GlobalFlags.Colors (wrong spelling btw as company policy is British English) became authoritative setting of determining whether to display colours. The call to `terminal.Colours() bool` already gives an educated guess whether colours should be enabled (by examining if the stdout is a tty). 

Discussed this with @moitias and one possibility is that GlobalFlags.Colous would be removed and we'd introduce two new booleans `ForceColours` and `DisableColours` with relevant flags. We would then only call terminal.ForceColours should one of the flags be set and otherwise let automatic detection of terminal enable the colours.

When it comes to logging I would suggest going without any colours at all especially if we log to stderr.

Reply:
:+1: these have been now implemented in the head of the branch.